stubs/jwcrypto/jwcrypto/jwk.pyi

```diff
@@ -29,10 +29,10 @@ priv_bytes: Incomplete
 JWKTypesRegistry: Incomplete
 
 class ParmType(Enum):
```

Comment:
Unrelated but I'm pretty sure this is a typo for "Param" in the runtime package.